server/src/main/java/timely/api/request/VersionRequest.java

```diff
@@ -0,0 +1,39 @@
+package timely.api.request;
+
+import io.netty.handler.codec.http.QueryStringDecoder;
+import timely.api.annotation.Http;
+import timely.api.annotation.Tcp;
+import timely.api.annotation.WebSocket;
+
+@Tcp(operation = "version")
+@Http(path = "/version")
+@WebSocket(operation = "version")
+public class VersionRequest implements TcpRequest, HttpGetRequest, HttpPostRequest, WebSocketRequest {
+
+    public static final String VERSION = "0.0.2";
```

Comment:
Maven packaging might put the version into the jar manifest. Might be able to read it from there with something like: 
```
getClass().getPackage().getImplementationVersion()
```